docs/source/storage/general_storage.rst

```diff
@@ -1,34 +1,32 @@
 .. _general_storage:
 
+.. |ss| raw:: html
+
+    <strike>
+
+.. |se| raw:: html
+
+    </strike>
+
 *****************************
 General Data Storage Options
 *****************************
 The Research Computing (RC) team is responsible for the procurement and ongoing
 maintenance of several data storage options, including active and archive
 storage solutions. If you are affiliated with Northeastern, you can request
-one or more storage solutions to meet your storage needs. It is highly recommended
-that if you anticipate needing storage as part of a grant requirement,
-schedule a consultation with a Research Computing staff member to understand what
+one or more storage solutions to meet your storage needs. If you anticipate needing storage as part of a grant requirement,
+please schedule a consultation with an RC staff member to understand what
 storage options would best meet your research needs.
 You can schedule an online consultation on the `RC website <https://rc.northeastern.edu/support/consulting>`_.
 
 .. note::
    If you have an account on Discovery, see :ref:`discovery_storage`
-   for details on the storage available to you specifically for use with Discovery's compute resources. The options listed below are not connected to Discovery.
-
-**NAME:** ``/research``
-  - **DESCRIPTION:** This storage tier is intended to be a repository for data derived from equipment such as lab machines,
-    instruments, etc. The performance capabilities are not intended for parallel or high performance workloads.
-    Data are backed up and a second copy is created. You can request storage on ``/research`` by submitting a `New Storage Space request <https://bit.ly/NURC-NewStorage>`_.
-    See the section "Connecting to /research" below for information on how to connect to this storage.
-  - **QUOTA:** Each group can request up to **35TB** of free storage across all supplemental storage tiers: ``/work/<groupname>``, ``/research`` () and ``/nese``.  
-
+   for details on the storage available to you specifically for use with Discovery's compute resources. The option listed below is not connected to Discovery.
 
 **NAME:** ``/nese``
```

Comment:
I feel like this section is the main purpose of this page, but with the Note and important boxes, this section gets a little lost in the middle. Can we just make it so that the first "note" is just a second paragraph? 

Although, now that I'm looking at it again, would it be best to just combine these two pages? Especially since there is only one option left? We could then take out the first note completely. (And the note at the bottom of the "Storage Accessible on Discovery" page that still references /research.)

Reply:
@tketchem14 - I think you are right because this page only has one option left, I think it would make sense to combine it with the first page. Do you want me to combine the pages to see how it appears and for adjustments prior to removing this page from this PR?